lib/matplotlib/container.py

```diff
@@ -123,8 +123,16 @@ def __init__(self, lines, has_xerr=False, has_yerr=False, **kwargs):
 
 
 class StemContainer(Container):
-
     def __init__(self, markerline_stemlines_baseline, **kwargs):
+        '''
+        Parameters
+        ----------
+        markerline_stemlines_baseline : tuple
+            Tuple of ``(markerline, stemlines, baseline)``.
+            ``markerline`` contains the `LineCollection` of the markers,
+            ``stemlines`` is a list of the `Line2D` of each line,
```

Comment:
Should this doc entry be updated to say LineCollection? Does it matter? Should we coerce to one type or another?